marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -1306,6 +1308,7 @@ const Flows: React.FC<Props> = ({
         let viewStepId = `${flowName}-${stepNumber}`;
         let stepDefinitionType = step.stepDefinitionType ? step.stepDefinitionType.toLowerCase() : "";
         let stepDefinitionTypeTitle = StepDefinitionTypeTitles[stepDefinitionType];
+        let stepJopDetail = latestJobData && latestJobData[flowName] && latestJobData[flowName] ? latestJobData[flowName].find(el => el.stepId === step.stepId) : null;
```

Comment:
Minor thing here but can we fix the variable name? Assuming 'stepJobDetail' is intended.